src/view/rooteditableelement.js

```diff
@@ -35,6 +35,17 @@ export default class RootEditableElement extends EditableElement {
 		 * @member {String}
 		 */
 		this.rootName = 'main';
+
+		/**
+		 * A DOM attributes cache. It saves the initial values of DOM attributes before the DOM element
+		 * is {@link module:engine/view/view~View#attachDomRoot attached} to the view so later on, when
+		 * the view is destroyed, they can be easily restored. This way, the DOM element can go back to
+		 * the (clean) state as if the editing view never used it.
+		 *
+		 * @protected
+		 * @member {Object.<String,String>}
+		 */
+		this._initialDomAttributes = {};
```

Comment:
I think it is a wrong place to store this. This is `attachDomRoot` internal logic and should be stored in a `WeakMap` inside `view/view.js`, not as a `RootEditableElement` property. This is a property which is not used by `RootEditableElement` class at all.